src/test/features/response/routes/reject-all-of-claim.ts

```diff
@@ -143,7 +143,21 @@ describe('Defendant response: full admission options', () => {
               .set('Cookie', `${cookieName}=ABC`)
               .send({ option: 'alreadyPaid' })
               .expect(res => expect(res).to.be.redirect
-                .toLocation(ResponsePaths.taskListPage
+                .toLocation(ResponsePaths.defendantHowMuchPaidClaimant
+                  .evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })))
+          })
+
+          it('should redirect to send your response by email page when counterclaim option is selected', async () => {
+            claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+            draftStoreServiceMock.resolveFind('response', draftOverride)
+            draftStoreServiceMock.resolveSave()
+
+            await request(app)
+              .post(pagePath)
+              .set('Cookie', `${cookieName}=ABC`)
+              .send({ option: 'counterClaim' })
```

Comment:
Use `RejectAllOfClaimOption.COUNTER_CLAIM` as a value to make refactoring easier.